fga/model/model.fga

```diff
@@ -37,18 +39,52 @@ type organization
     define can_edit: [service] or admin or owner or can_edit from parent
     define can_view: [service] or member or admin or owner or can_edit or can_view from parent
     # additional fine-grained permissions
-    ## allow owner and assigned users to view audit logs
+    # allow owner and assigned users to view audit logs
     define audit_log_viewer: [user] or owner or audit_log_viewer from parent
-    ## allow members to invite other members
+    # allow members to invite other members
     define can_invite_members: [user] or member or can_edit or can_invite_members from parent
-    ## only allow users with edit access to the org to invite other admins
+    # only allow users with edit access to the org to invite other admins
     define can_invite_admins: [user] or can_edit or can_invite_admins from parent
+    # object creation permissions
+    # these permissions should all be defined as:
+    #     define [object]_creator: [group#member]
+    #     define can_create_[object]: can_edit or [object]_creator
+
+    define control_creator: [group#member]
+    define can_create_control: can_edit or control_creator
+
+    define control_objective_creator: [group#member]
+    define can_create_control_objective: can_edit or control_objective_creator
+
+    define group_creator: [group#member]
+    define can_create_group: can_edit or group_creator
+
+    define internal_policy_creator: [group#member]
+    define can_create_internal_policy: can_edit or internal_policy_creator
+
+    define narrative_creator: [group#member]
+    define can_create_narrative: can_edit or narrative_creator
+
+    define procedure_creator: [group#member]
+    define can_create_procedure: can_edit or procedure_creator
+
+    define program_creator: [group#member]
+    define can_create_program: can_edit or program_creator
+
+    define risk_creator: [group#member]
+    define can_create_risk: can_edit or risk_creator
+
+    define template_creator: [group#member]
+    define can_create_template: can_edit or template_creator
```

Comment:
it looks like you hit all of the main "compliance" related objects but should we also add the other existing objects that don't yet have these types of permission structures? (e.g. webhooks, hush / secrets, entity, etc.?)

Reply:
yes - these will get added when I go back through and add additional permissions (those objects dont even have basic permissions yet and didnt want to make this PR even larger. Going to do a final sweep of all schemas next to add the rest of the permissions